src/WebJobs.Script/Diagnostics/Extensions/DescriptionLoggerExtension.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.Diagnostics.Extensions
+{
+    public static class DescriptionLoggerExtension
+    {
+        private static readonly Action<ILogger, string, string, Exception> _assemblyDynamicluResolved =
+        LoggerMessage.Define<string, string>(LogLevel.Trace, new EventId(1, nameof(AssemblyResolved)),
```

Comment:
Should this be "AssemblyDynamiclyResolved"? Also, should the above be "_assemblyDynamiclyResolved"?